src/main/java/core/basesyntax/dao/FruitsDaoImpl.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+
+public class FruitsDaoImpl implements FruitsDao {
```

Comment:
```suggestion
public class FruitDaoImpl implements FruitDao {
```
same don't need to define it in plural